lib/ui/painting.dart

```diff
@@ -4452,10 +4452,25 @@ enum TileMode {
   decal,
 }
 
+Float32List _encodeWideColorList(List<Color> colors) {
```

Comment:
I remember talking about preserving the CS, but I'm guessing that gradients are going to want to normalize to a similar CS for lerping - and extendedSRGB is probably the best for fractional combinations?

Reply:
We do preserve the colorspace from the users perspective.  The conversion to extended srgb only happens when communicating the values to the engine.